src/main/java/tracko/model/items/Quantity.java

```diff
@@ -0,0 +1,42 @@
+package tracko.model.items;
+
+import static tracko.commons.util.CollectionUtil.requireAllNonNull;
+
+/**
+ * Represents the quantity of an item.
+ */
+public class Quantity {
+    public final int quantity;
+
+    /**
+     * Constructs a {@code Quantity}.
+     * @param quantity The quantity of an item.
+     */
+    public Quantity(int quantity) {
```

Comment:
I think Quantity can be validated as not being a negative integer here

Reply:
That's a good catch! Thank you, I fixed it